contrib/excluder/excluder-template

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# This script will either enable, or disable, the updating 
+#  of openshift rpms via yum
+#
+##############
+# VARIABLES
+##############
+CONF_FILE="@@CONF_FILE-VARIABLE@@"
+PACKAGE_LIST="@@PACKAGE_LIST-VARIABLE@@"
+
+##############
+# HELP
+##############
+function usage() {
+  {
+  echo "Usage: ${0} <exclude | unexclude>"
+  echo
+  echo "exclude | disable"
+  echo "  exclude packages via an excludes= line in ${CONF_FILE}"
+  echo "  This disables updates for ${PACKAGE_LIST} packages"
+  echo
+  echo "unexclude | enable"
+  echo "  removes ${PACKAGE_LIST} from the excludes= line in ${CONF_FILE}"
+  echo "  This re-enables updates for packages"
+  echo
+  } 1>&
+  exit 1
+}
+
+##############
+# FUNCTIONS
+##############
+function exclude_packages() {
+  # Ensure there is an exclude line
+  if ! grep -q exclude= "${CONF_FILE}" ; then
+    echo "Adding exclude= to ${CONF_FILE}"
+    sed -i 's|\(obsoletes=.*\)|\1\nexcludes=|' "${CONF_FILE}"
+  fi
+  # Add each package to the exclude line
+  for package in ${PACKAGE_LIST}
+  do
+    if ! grep exclude= "${CONF_FILE}" | grep -F -q " ${package} " ; then
```

Comment:
Hmm. The doc on `-F` is about splitting on newlines:
```
       -F, --fixed-strings
              Interpret PATTERN as a list of fixed strings, separated by newlines, any of which is to be matched.
```

Let me think, this is weird behavior.